perllib/FixMyStreet/App/Controller/Dashboard.pm

```diff
@@ -256,18 +274,28 @@ sub generate_grouped_data : Private {
         %grouped = map { $_->category => {} } @{$c->stash->{contacts}};
     }
     my $problems = $c->stash->{objects_rs}->search(undef, {
-        group_by => [ map { ref $_ ? $_->{-as} : $_ } @groups ],
-        select   => [ @groups, { count => 'me.id' } ],
-        as       => [ @groups == 2 ? qw/key1 key2 count/ : qw/key1 count/ ],
+        group_by => [ map { ref $_ ? $_->{-as} : $_ } (@groups, 'me.extra') ],
```

Comment:
I'd fetch only the extra->>'group' specifically here, rather than all of extra - `extra` will be unique for each report pretty much if there's any extra data at all, thereby stopping the group by working at all, I'd assume.

Reply:
Thanks for chat - this should be done now